### ISSUE TITLE
Can't validate JSON list response

### ISSUE THREAD

**Issue by WhileLoop** [taverntesting/tavern#7]
Validating this response:

```
[
    {
        "author": "Dan Brown",
        "isbn": "123456",
        "title": "Digital Fortress"
    },
    {
        "author": "JK Rowling",
        "isbn": "234567",
        "title": "Harry Potter and the Chamber of Secrets"
    }
]
```

With this test:

```
test_name: json list response

stages:
  - name: get list
    request:
      url: http://localhost:8080/
      method: GET
    response:
      status_code: 200
      body:
        - author: Dan Brown
          isbn: '123456'
          title: Digital Fortress
        - author: JK Rowling
          isbn: '234567'
          title: Harry Potter and the Chamber of Secrets
```

Gives this error:

```
2017-11-16 20:15:17,978 [ERROR]: (tavern.schemas.files:38) Error validating /var/folders/6p/d1m45dr1031800ldbk945jrc0000gn/T/tmpes45ny99.yaml
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/tavern/schemas/files.py", line 36, in verify_generic
    verifier.validate()
  File "/usr/local/lib/python3.6/site-packages/pykwalify/core.py", line 176, in validate
    error_msg=u'.\n - '.join(self.validation_errors)))
pykwalify.errors.SchemaError: <SchemaError: error code 2: Schema validation failed:
 - Value '[{'author': 'Dan Brown', 'isbn': '123456', 'title': 'Digital Fortress'}, {'author': 'JK Rowling', 'isbn': '234567', 'title': 'Harry Potter and the Chamber of Secrets'}]' is not a dict. Value path: '/stages/0/response/body'.: Path: '/'>
```

**Comment by michaelboulton** [comment]
I'm looking at implementing this now with the caveat that the items in the `body` have to be the same order as returned from the API.

This isn't really an issue for small tests like this - the order of list items does matter in JSON/YAML - but it might cause issues with testing more complicated APIs that just return a load of items in a random order (eg, farming out asynchronous tasks that each return a value and add it to a list in whatever order they get processed in).

**Comment by michaelboulton** [comment]
Should be fixed in 0.1.2
